website/data/docs-nav-data.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "title": "Introduction",
+    "title": "What is Waypoint?",
```

Comment:
Looks like you need to rebase. There are a few changes in the `docs-nav-data.json` file that are being reverted. I dealt with this with my backport PR. I believe the only changes needed in this file are the use case `title` & `path` being added.